modules/gltf/src/lib/extensions/KHR_techniques_webgl.js

```diff
@@ -0,0 +1,58 @@
+// GLTF EXTENSION: KHR_techniques_webgl
+// https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_techniques_webgl
+/* global TextDecoder */
+import GLTFScenegraph from '../gltf-scenegraph';
+import {KHR_TECHNIQUES_WEBGL} from '../gltf-constants';
+
+function resolveTechniques({programs = [], shaders = [], techniques = []}, gltfScenegraph) {
+  const textDecoder = new TextDecoder();
+
+  shaders.forEach(shader => {
+    if (Number.isFinite(shader.bufferView)) {
+      shader.code = textDecoder.decode(
+        gltfScenegraph.getTypedArrayForBufferView(shader.bufferView)
```

Comment:
Interesting that they stuck the text shaders in buffers rather than just put them in strings in the JSON...

Reply:
yup